### PR TITLE
Moved files and updated file references

### LINKED ISSUE #11: Update image naming scheme and document it
The current image naming scheme is messy and should be updated and documented, so that all future features which require the use of images are done consistently.

Suggestion is to create a new folder under images for each character (for example, Mario and Luigi)

### LINKED ISSUE #12: Remove 20 image hard-coded loop
Loop is currently hard-coded for the number of images to use to train the ML algorithm. Once #11 is implemented, it should be as easy as running a foreach loop over the entire folder, as we won't need to manually find files.

### PR DESCRIPTION
**Pull request by BenDew21**
Files moved to their own folders in the images folder (documented in README.md), based on their category. In addition, file location references have been updated, and the hard-coded loop removed. This addresses #11 and #12

### CODE REVIEW

**Comment by JoanneLin168** [comment]
Seems good, will wait until @karikowska has had a look before I merge it

**Review by JoanneLin168 — APPROVED** [review]
Code seems a lot better than what was previously there - will help a lot in the future

**Comment by karikowska** [comment]
Thank you for the much needed changes! Approved.

**Pull request merged**
